gradle-baseline-java/src/main/java/com/palantir/baseline/plugins/BaselineErrorProne.java

```diff
@@ -152,6 +124,152 @@ private static void configureErrorProneOptions(ErrorProneOptions errorProneOptio
                 // seamless rollout.
                 "SuperCallToObjectMethod");
 
+        // These checks are default disabled in error-prone. However, in error-prone 2.37.0 a bug has caused them
+        // to be enabled by default. So we're disabling them here until error-prone is fixed, then these can all
+        // be removed.
+        // See https://github.com/google/error-prone/issues/4943
+        errorProneOptions.disable(
```

Comment:
Could we reference `BuiltInCheckerSuppliers.DISABLED_CHECKS` here instead of copying this list?